### PR TITLE
{CI} Doc's feedback GitHub issue template

### PR DESCRIPTION
**Pull request by dbradish-microsoft**
We need an additional GitHub issue template in azure-cli for customer feedback that is recieved on autogenerated reference docs. This template aligns with the Azure PowerShell template found here: https://github.com/Azure/azure-powershell/blob/main/.github/ISSUE_TEMPLATE/4-AZ-DOC-BUG.yml.

Anytime a customer uses the docs or product feedback links in MicrosoftDocs/azure-docs-cli WHEN IN AN AUTOGENERATED REFERENCE DOC, they will be taken to this new template. This will keep MicrosoftDocs/azure-docs-cli GitHub issues about conceptual content, and reference feedback is channeled correctly to the engineering teams.

**MicrosoftDocs/azure-docs-cli [PR 4788](https://github.com/MicrosoftDocs/azure-docs-cli/pull/4788/files#diff-25e6592e0a824906d8ada0a84573256efbdaba49bee6833df043caaa2ca91129) is waiting on this PR to merge.**

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: .github/ISSUE_TEMPLATE/docs_feedback.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    label: Author
+    description: GitHub Id of the author
+- type: input
+  id: documentVersionIndependentId
+  attributes:
+    label: Document Id
+- type: markdown
+  attributes:
+    value: >
+      The Azure CLI team is listening, please let us know how we are doing: https://aka.ms/azpssurvey?Q_CHL=REPO.
```
Comment (sdwheeler) [inline .github/ISSUE_TEMPLATE/docs_feedback.yml:69]: Is this the correct URL for Azure CLI?

Comment (dbradish-microsoft) [reply .github/ISSUE_TEMPLATE/docs_feedback.yml:69]: Good catch. Thank you, Sean.

**Review by AllyW — APPROVED** [review]

**Pull request closed without merging**